examples-nosql-dotnet-sdk/sqlexamples/TableJoins.cs

```diff
@@ -0,0 +1,171 @@
+                                                         flightDate TIMESTAMP(4),
+                                                         fltRouteSrc STRING,
+                                                         fltRouteDest STRING,
+                                                         estimatedArrival TIMESTAMP(4),
+                                                         actions JSON,
+                                                         primary key(flightNo))";
+      private const string data1=@"{""ticketNo"": ""1762344493810"",""confNo"" : ""LE6J4Z"" }";
+      private const string data2=@"{""ticketNo"":""1762344493810"",
+                                    ""id"":""79039899165297"",
+                                    ""tagNum"":""17657806255240"",
+                                    ""routing"":""MIA/LAX/MEL"",
+                                    ""lastActionCode"":""OFFLOAD"",
+                                    ""lastActionDesc"":""OFFLOAD"",
+                                    ""lastSeenStation"":""MEL"",
+                                    ""lastSeenTimeGmt"":""2019-02-01T16:13:00Z"",
+                                    ""bagArrivalDate"":""2019-02-01T16:13:00Z""}";
+      private const string data3=@"{""ticketNo"":""1762344493810"",
+                                    ""id"":""79039899165297"",
+                                    ""flightNo"":""BM604"",
+                                    ""flightDate"":""2019-02-01T06:00:00Z"",
+                                    ""fltRouteSrc"":""MIA"",
+                                    ""fltRouteDest"":""LAX"",
+                                    ""estimatedArrival"":""2019-02-01T11:00:00Z"",
+                                    ""actions"":[ {
+                                       ""actionAt"" : ""MIA"",
+                                       ""actionCode"" : ""ONLOAD to LAX"",
+                                       ""actionTime"" : ""2019-02-01T06:13:00Z""
+                                    }, {
+                                       ""actionAt"" : ""MIA"",
+                                       ""actionCode"" : ""BagTag Scan at MIA"",
+                                       ""actionTime"" : ""2019-02-01T05:47:00Z""
+                                    }, {
+                                       ""actionAt"" : ""MIA"",
+                                       ""actionCode"" : ""Checkin at MIA"",
+                                       ""actionTime"" : ""2019-02-01T04:38:00Z""} ]}";
+      private const string stmt1 ="SELECT * FROM ticket a LEFT OUTER JOIN ticket.bagInfo.flightLegs b ON a.ticketNo=b.ticketNo";
+
+      //Get a connection handle for Oracle NoSQL Database Cloud Service
+      private async static Task<NoSQLClient> getconnection_cloud()
+      {
+         //replace the place holder for compartment with your region identifier and OCID of your compartment
+         var client =  new NoSQLClient(new NoSQLConfig
```

Comment:
the comment is only referring to compartment.   there are 2 place holders.  This should be ..   replace the region and compartment place holders with your ....